layouts/community/ortho_4x12/neo2marin/keymap.c

```diff
@@ -0,0 +1,221 @@
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  , ____  , ____  , ____  , ____  , ____  ,      ____  , ____  , ____  , ____  , ____  , ____
+  //`-------+-------+-------+-------+-------+-------'    `-------+-------+-------+-------+-------+-------'
+  ),
+
+  [L_MOV] = LAYOUT_ortho_4x12(
+  //,-------+-------+-------+-------+-------+-------.    ,-------+-------+-------+-------+-------+-------.
+      ____  ,KC_PGUP,KC_BSPC, KC_UP ,KC_DEL, KC_PGDN,      ____  , KC_7  , KC_8  , KC_9  ,DE_PLUS,DE_MINS,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  ,KC_HOME,KC_LEFT,KC_DOWN,KC_RGHT,KC_END ,      ____  , KC_4  , KC_5  , KC_6  ,DE_COMM, DE_DOT,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  ,KC_ESC ,KC_TAB ,KC_INS, KC_ENTER,____  ,      KC_0  , KC_1  , KC_2  , KC_3  ,DE_SCLN, ____  ,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  , ____  , ____  , ____  , ____  , ____  ,      ____  , KC_0  , ____  , ____  , ____  , ____ 
+  //`-------+-------+-------+-------+-------+-------'    `-------+-------+-------+-------+-------+-------'
+  ),
+
+  [L_FUN] = LAYOUT_ortho_4x12(
+  //,-------+-------+-------+-------+-------+-------.    ,-------+-------+-------+-------+-------+-------.
+      ____  , AU_ON ,KC_VOLD,ME_MAXM,KC_VOLU, WURST ,      MAC1R , KC_F7 , KC_F8 , KC_F9 , KC_F12,KC_PSCR,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  ,KC_MSTP,KC_MPRV,KC_MPLY,KC_MNXT, NEO2  ,      MACRS , KC_F4 , KC_F5 , KC_F6 , KC_F11,KC_BRK ,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  , AU_OFF,KC_TAB ,KC_INS ,KC_ENTER,QWERTZ,      MAC1P , KC_F1 , KC_F2 , KC_F3 , KC_F10, ____  ,
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+      ____  , ____  , ____  , ____  , ____  , ____  ,      ____  , ____  , ____  , ____  , ____  , ____
+  //|-------+-------+-------+-------+-------+-------|    |-------+-------+-------+-------+-------+-------|
+  ),
+
+};
+
+#ifdef AUDIO_ENABLE
+float tone_qwertz[][2]   = SONG(QWERTY_SOUND);
+float tone_neo2[][2]     = SONG(DVORAK_SOUND);
+#endif
+
+void persistent_default_layer_set(uint16_t default_layer) {
+  eeconfig_update_default_layer(default_layer);
+  default_layer_set(default_layer);
+}
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTZ:
+      if (record->event.pressed) {
+        #ifdef AUDIO_ENABLE
+          PLAY_SONG(tone_qwertz);
+        #endif
+        persistent_default_layer_set(1UL<<L_QWZ);
```

Comment:
This is all handled by simple function, now.

https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer


```suggestion
        set_single_persistent_default_layer(L_QWZ);
```

Reply:
Thank you for pointing me at that function!
Still, I decided I do not want to make the default layer persistent between reboots.
So I changed the code to `default_layer_set();` and `PLAY_SONG();` instead.
Unfortunately, the `DF()` key macro does not play back a sound, so I still have to use these custom keys.